Transbank/PatpassRest/Common/Detail.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Transbank.PatpassRest.Common
```

Comment:
I think that the namespace should be `Transbank\Patpass\PatpassByWebpay` as Felipe describe here https://continuumhq.slack.com/archives/C56UAPE3S/p1564088498019900?thread_ts=1564088489.019800&cid=C56UAPE3S